iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/storable_function.hpp

```diff
@@ -65,9 +65,17 @@ class storable_function<StorageType, signature<ReturnType, Args...>>
               typename = typename std::enable_if<std::is_class<Functor>::value
                                                      && is_invocable_r<ReturnType, Functor, Args...>::value,
                                                  void>::type>
+    /// @NOLINTJUSTIFICATION the storable function should implicitly behave like any generic constructor, adding
```

Comment:
Same as mentioned [here](https://github.com/eclipse-iceoryx/iceoryx/pull/1495#discussion_r920005056), the `@` can be removed.

```suggestion
    /// NOLINTJUSTIFICATION the storable function should implicitly behave like any generic constructor, adding
```